### PR TITLE
479 refactoring menu icons

### LINKED ISSUE #479: Refactoring Menu Icons
Change all icons in our Menu Screens: 

1_ Activity Log - Connections - Transactions - News 

2_Trust People- Directly trusted - In your network - External 

3_ Someones profile - Activity- Mutually Trusted

Design Material: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=7373%3A13457

### LINKED ISSUE #570: Badge and notification logic + Trust activity is not opened when entering activity log
**Describe the bug**
When a trust event has happened, such as
- trusting or untrusting another account
- being trusted or untrusted by another account
it triggers a badge in the activity log bell.
Upon entering the activity log, the user is referred to the transaction log where nothing has changed. 
The connections log is not opened automatically and shows no badge

**To Reproduce**
1. Enter the activity log first if there are unopened notifications
2. Go to another account and trust or untrust it
3. Go back to the dashboard and wait for the loading wheel to stop spinning on the bell icon
4. See a badge with a 1 (1 new notification)
5. Enter the activity log by clicking the bell
6. Experience the "bug" above
7. Enter the connections tab and see a dot indicating a new notification that wasn't evident before explicitly switching tab.

**Expected behavior**
When entering the activity log I would expect the tab with the highest number to open automatically. If they are even (2 of each for example), start from the left. In the case above this would cause the connections tab to open. If there are notification in multiple tabs I expect the other (non-opened) tab(s) to have badges with numbers of unread notifications. Similar to how badges show in search tabs.

**Device**
All devices.

**Additional context**
Use search view badges as inspiration. Make sure figma design is up to date.

### PR DESCRIPTION
**Pull request by louilinn**
Closes #479 

- [x] replace icons in activity log
- [x] replace icons in search view
- [x] replace icons in profile view
- [x] update active color scheme of tab navigation
- [x] update inactive color scheme
- [x] implement hover behavior for tabs navigation with colors
- [x] change badge color
- [x] replace illustration for no search results found

Also closes #570

### CODE REVIEW

**Comment by louilinn** [comment]
Pleeease use rebase!! <3 (not merge)

File: src/components/Finder.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {
-  Badge,
+  // Badge,
```
Comment (louilinn) [inline src/components/Finder.js]: remove

File: src/styles/icons.js [hunk 2]
```diff
@@ -291,6 +303,8 @@ export const iconSelector = (icon) => {
       return IconTriangleWarning;
     case 'IconTrustActive':
       return IconTrustActive;
+    case 'IconTrustedDirectly':
+      return IconTrustedDirectly;
```
Comment (louilinn) [inline src/styles/icons.js:309]: Make sure you add all icons, you did not include "IconNoSearchResult"

File: src/components/Finder.js [hunk 3]
```diff
@@ -80,9 +86,22 @@ const useStyles = makeStyles((theme) => ({
       borderBottom: `2px solid ${theme.palette.primary.main}`,
     },
   },
-  iconTrustActice: {
-    '& path': {
-      fill: 'inherit',
+  tabNavigationMain: {
```
Comment (louilinn) [inline src/components/Finder.js]: This styling has to do with tab navigation (tab navigation action) and should be in that component. The styling is missing in activity log

File: src/views/Profile.js [hunk 4]
```diff
@@ -51,10 +52,25 @@ const DEFAULT_PANEL = PANEL_TRUST;
...
+const useStyles = makeStyles((theme) => ({
   mutualUserCard: {
     cursor: 'pointer',
   },
+  iconActive: {
+    '& path': {
+      fill: theme.custom.colors.violet,
+    },
+  },
+  badgeContainer: {
```
Comment (louilinn) [inline src/views/Profile.js]: Duplicate styling and redundant.
This styling is already in Badge component?

File: src/views/Profile.js [hunk 5]
```diff
@@ -221,8 +240,23 @@ const ProfileContent = ({ address, deploymentStatus, trustStatus }) => {
           value={PANEL_ACTIVITY}
         />
         <TabNavigationAction
-          icon={<IconFriends />}
-          label={translate('Profile.bodyTrustedBy')}
+          icon={
+            <Badge
```
Comment (louilinn) [inline src/views/Profile.js]: Use BadgeCircle!

File: src/components/BadgeCircle.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
+import { Badge } from '@material-ui/core';
```
Comment (louilinn) [inline src/components/BadgeCircle.js:1]: We should check behaviour of badge with 2-digit numbers

**Review by louilinn — COMMENTED** [review]
Screenshot from 2023-02-15 22-15-30
Screenshot from 2023-02-15 21-57-34

Main tab styling should be on tab component. Styling is wrong in one view as a result of it being in the wrong place. Some other comments too.

Badge numbers also doesn't seem to be centered in height

Changes requested :)

**Comment by louilinn** [comment]
Once you are done, I will have to redo this branch using cherry-pick and force push due to the merge commits. Follow the rebase instructions next time. I can show you if needed. 

Do not merge.

**Comment by louilinn** [comment]
I cherry-picked from here: https://github.com/CirclesUBI/circles-myxogastria/pull/594/files
As I thought this was a good place to include that change as I needed to do half the change to fix the issue from the pictures above. https://github.com/CirclesUBI/circles-myxogastria/pull/594/files #570 

After talking to product we can adjust notification behavior in this PR before merging. This branch is in preview.

File: src/components/BadgeCircle.js (hunk 6 above)
Comment (mikozet) [reply src/components/BadgeCircle.js:1]: You can edit html and have a look how it looks like. We can apply different styles for two digits or we can enlarge circle as well so it fit better once there are two digits..

**Comment by mikozet** [comment]
> Main tab styling should be on tab component. Styling is wrong in one view as a result of it being in the wrong place. Some other comments too.
> 
> Badge numbers also doesn't seem to be centered in height

Corrected

**Comment by louilinn** [comment]
What I mean is that the number in the badge is quite high inside the circle
Screenshot from 2023-02-16 09-36-45

I think we should probably use the standard badge everywhere. This one doesn't adapt to content
Screenshot from 2023-02-16 09-41-03

**Comment by louilinn** [comment]
Deployed in preview.

Includes two small fixes from product feedback too:
Closes: #609 

After this PR rafactoring-2022 should be r4r.

**Comment by louilinn** [comment]
Product is happy with visuals now. Only code review needed.

**Review by mikozet — APPROVED** [review]

**Pull request merged**